<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="8" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="31328" started-at="2018-06-27T20:34:23Z" finished-at="2018-06-27T20:34:55Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="31328" started-at="2018-06-27T20:34:23Z" finished-at="2018-06-27T20:34:55Z">
      <class name="com.ecom.qa.testcases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="setUp" is-config="true" duration-ms="6340" started-at="2018-06-27T20:34:23Z" finished-at="2018-06-27T20:34:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="homePageLogin()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="homePageLogin" duration-ms="3999" started-at="2018-06-27T20:34:30Z" finished-at="2018-06-27T20:34:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="tearDown" is-config="true" duration-ms="95" started-at="2018-06-27T20:34:34Z" finished-at="2018-06-27T20:34:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="setUp" is-config="true" duration-ms="5402" started-at="2018-06-27T20:34:35Z" finished-at="2018-06-27T20:34:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="homePageLogout()[pri:1, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="homePageLogout" duration-ms="6212" started-at="2018-06-27T20:34:40Z" finished-at="2018-06-27T20:34:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageLogout -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="tearDown" is-config="true" duration-ms="90" started-at="2018-06-27T20:34:46Z" finished-at="2018-06-27T20:34:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="setUp" is-config="true" duration-ms="5633" started-at="2018-06-27T20:34:47Z" finished-at="2018-06-27T20:34:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="quickView()[pri:2, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="quickView" duration-ms="1289" started-at="2018-06-27T20:34:52Z" finished-at="2018-06-27T20:34:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quickView -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ecom.qa.testcases.LoginPageTest@6eceb130]" name="tearDown" is-config="true" duration-ms="102" started-at="2018-06-27T20:34:54Z" finished-at="2018-06-27T20:34:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ecom.qa.testcases.LoginPageTest -->
      <class name="com.ecom.qa.testcases.ApiCallsTest">
        <test-method status="PASS" signature="apiTestReadUsersList001()[pri:0, instance:com.ecom.qa.testcases.ApiCallsTest@10a035a0]" name="apiTestReadUsersList001" duration-ms="638" started-at="2018-06-27T20:34:34Z" finished-at="2018-06-27T20:34:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apiTestReadUsersList001 -->
        <test-method status="FAIL" signature="apiTestReadUsersList002()[pri:1, instance:com.ecom.qa.testcases.ApiCallsTest@10a035a0]" name="apiTestReadUsersList002" duration-ms="409" started-at="2018-06-27T20:34:46Z" finished-at="2018-06-27T20:34:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Eve is found in response **Expected failure**]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Eve is found in response **Expected failure**
at org.testng.Assert.fail(Assert.java:96)
at com.ecom.qa.testcases.ApiCallsTest.apiTestReadUsersList002(ApiCallsTest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apiTestReadUsersList002 -->
        <test-method status="PASS" signature="apiTestCreateUser003()[pri:2, instance:com.ecom.qa.testcases.ApiCallsTest@10a035a0]" name="apiTestCreateUser003" duration-ms="409" started-at="2018-06-27T20:34:54Z" finished-at="2018-06-27T20:34:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apiTestCreateUser003 -->
        <test-method status="PASS" signature="apiTestUpdateUser004()[pri:3, instance:com.ecom.qa.testcases.ApiCallsTest@10a035a0]" name="apiTestUpdateUser004" duration-ms="323" started-at="2018-06-27T20:34:54Z" finished-at="2018-06-27T20:34:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apiTestUpdateUser004 -->
        <test-method status="PASS" signature="apiTestDeleteUser005()[pri:4, instance:com.ecom.qa.testcases.ApiCallsTest@10a035a0]" name="apiTestDeleteUser005" duration-ms="340" started-at="2018-06-27T20:34:54Z" finished-at="2018-06-27T20:34:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apiTestDeleteUser005 -->
      </class> <!-- com.ecom.qa.testcases.ApiCallsTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
